src/raiden_libs/register_service.py

```diff
@@ -65,48 +82,86 @@ def main(
 
     # Register if not yet done
     if current_deposit <= 0:
```

Comment:
is that the stake of the service? then maybe call it stake?

Reply:
Good point, I wonder how that actually worked. There is no field `deposits` any more?!